api/app/views.py

```diff
@@ -1,6 +1,56 @@
-from django.http import JsonResponse
+import json
+
+from django.conf import settings
+from django.contrib.staticfiles.storage import staticfiles_storage
+from django.http import HttpResponse, JsonResponse
+from django.template import loader
+
 from . import utils
 
 
 def version_info(request):
     return JsonResponse(utils.get_version_info())
+
+
+def index(request):
+    template = loader.get_template("webpack/index.html")
+
+    # if settings.DEBUG:
+    #    template = loader.get_template("index.html")
+
+    context = {
+        "linkedin_api_key": settings.LINKEDIN_API_KEY,
+    }
+
+    return HttpResponse(template.render(context, request))
+
+
+def project_overrides(request):
```

Comment:
What is this endpoint? Seems a bit skank. Maybe add a docstring to describe what it's doing if it has to stay...

Reply:
More please. Explain how it is used by the FE, i.e. essentially quoting @kyle-ssg's message below.